Extension/src/LanguageServer/client.ts

```diff
@@ -539,6 +540,8 @@ export interface TextDocumentWillSaveParams {
 interface InitializationOptions {
     packageVersion: string;
     extensionPath: string;
+    cacheStoragePath: string | undefined;
```

Comment:
Change `cacheStoragePath` to string. The object is not declared as optional on the language server changes so it does not need to be undefined here.